### PR TITLE
Add possibility to download pdf

### PR DESCRIPTION
**Pull request by otiala**
- Add possibility to download PDF from Respa Admin
- Add `share_of_tax` to receipt
- Add unit name to receipt

### CODE REVIEW

File: resources/views/receipt.py [hunk 1]
```diff
@@ -1,9 +1,43 @@
...
 from django.template.loader import get_template
+from django.utils.translation import ugettext_lazy as _
 from xhtml2pdf import pisa
 
+from respa_payments.integrations.paytrail_e2_utils import generate_order_number
+from respa_payments.models import Order
 
-def render_pdf_receipt_view(context):
+
+def render_pdf_receipt_view(request, order_id):
```
Comment (silenatte) [inline resources/views/receipt.py:12]: `request` parameter not used.

File: resources/models/reservation.py [hunk 2]
```diff
@@ -25,7 +25,7 @@
     get_dt, save_dt, is_valid_time_slot, humanize_duration, send_respa_mail,
     DEFAULT_LANG, localize_datetime, format_dt_range, build_reservations_ical_file,
 )
-from ..views.receipt import render_pdf_receipt_view
```
Comment (silenatte) [inline resources/models/reservation.py:28]: This deleted function is still used in this file.

Comment (otiala) [reply resources/models/reservation.py:28]: Changed

File: resources/views/receipt.py (hunk 1 above)
Comment (otiala) [reply resources/views/receipt.py:12]: It is. `request` is a required parameter in Django views.

Comment (silenatte) [reply resources/views/receipt.py:12]: But you are not passing request in when calling this function in some other place.

`receipt = render_pdf_receipt_view(receipt_context)`

Comment (silenatte) [reply resources/views/receipt.py:12]:
```
render_pdf_receipt_view: render_pdf_receipt_view
No value for argument 'order_id' in function callpylint(no-value-for-parameter)
```

Comment (otiala) [reply resources/views/receipt.py:12]: I understand the confusion, there is now `render_pdf_receipt_view`, which is a function-based view and `render_pdf_receipt`, which is just a function. See the newest commit, I changed the reference to the old function, and it should be working now.

File: resources/views/receipt.py [hunk 3]
```diff
@@ -1,9 +1,43 @@
...
+    reservation_period = '{0} - {1}'.format(
+        order.reservation.begin.strftime('%d.%m.%Y %H:%M'),
+        order.reservation.end.strftime('%d.%m.%Y %H:%M')
+    )
+    order_number = generate_order_number('VARAUS', order.reservation.resource.name, order.id)
+    reservation_name = '{0} ({1})'.format(
+        order.reservation.resource.name,
+        order.reservation.resource.unit.name
+    )
+    share_of_tax = str(round((order.sku.price / 100 * order.sku.vat), 2))
```
Comment (jjuvonen) [inline resources/views/receipt.py:28]: Bugi laskennassa.

**Pull request merged**